keywords/couchbaseserver.py

```diff
@@ -1026,6 +1026,43 @@ def get_collection_id(self, bucket, scope, collection):
                         col_id = collection_1["uid"]
         return col_id
 
+    def disable_replicas(self, bucket):
+        """ Disable replicas which needed for transaction app testing"""
+        data = {
+            "replicaNumber": 0
+        }
+        resp = self._session.post("{}/pools/default/buckets/{}".format(self.url, bucket), data=data)
+        print("response for the post call is ", resp)
```

Comment:
remove Print , since we have log. Remove the commented code

Reply:
print is not required . removed print and comments